code/src/Components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
consistency on the endings here` ; `put them on all or none . I struggle with the same cant decide 🤷‍♀️